GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/GarageController.cs

```diff
@@ -20,7 +20,7 @@ private void Awake()
     }
 
 
-    // fresh the page when on eable
+    // fresh the page when on enable
```

Comment:
Write comment as if you're writing a sentence. "Fresh ....". Apply this for all places please.

Reply:
got it